packages/kbn-optimizer/src/optimizer/optimizer_config.ts

```diff
@@ -33,14 +33,10 @@ import { findKibanaPlatformPlugins, KibanaPlatformPlugin } from './kibana_platfo
 import { getPluginBundles } from './get_plugin_bundles';
 import { filterById } from './filter_by_id';
 
-function pickMaxWorkerCount(dist: boolean) {
-  // don't break if cpus() returns nothing, or an empty array
-  const cpuCount = Math.max(Os.cpus()?.length, 1);
-  // if we're buiding the dist then we can use more of the system's resources to get things done a little quicker
-  const maxWorkers = dist ? cpuCount - 1 : Math.ceil(cpuCount / 3);
-  // ensure we always have at least two workers
-  return Math.max(maxWorkers, 2);
-}
+// don't break if cpus() returns nothing, or an empty array
+const CPU_COUNT = Math.max(Os.cpus()?.length, 1);
+// ensure we have at least three workers
+const MAX_WORKER_COUNT = Math.max(CPU_COUNT - 1, 3);
```

Comment:
I am concerned with this change and the effect it will have on dev machines and suggest we address in a separate PR. For context, running this on my machine results in my load going to 45+.

Reply:
Does it have any measurable impact on the performance of the optimizer? The load is only a problem if we're not getting better performance right?

Reply:
@tylersmalley I reverted the change